compiler/build.gradle

```diff
@@ -29,6 +30,6 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
 
     // The flag is needed because we extend an interface that uses @JvmDefault and the Kotlin
     // compiler requires this flag when doing so.
-    freeCompilerArgs += "-Xjvm-default=enable"
+    freeCompilerArgs += ["-Xjvm-default=enable", "-Xopt-in=com.squareup.anvil.annotations.ExperimentalAnvilApi"]
```

Comment:
Same here, I probably wouldn't mind adding this to the root file, but I can be convinced otherwise. What do you think?

Reply:
Ignore.